src/io/flutter/logging/FlutterLogEntryParser.java

```diff
@@ -83,25 +91,37 @@ public void onError(RPCError error) {
     }
   }
 
-  private FlutterDebugProcess debugProcess;
+  static class InspectorHelper {
```

Comment:
Is this helper class useful given when operating you typically need to handle only having a CompletableFuture<InspectorService.ObjectGroup> due to timing?

Reply:
Good point.  This class did more once but now doesn't have much reason to exist.

Removing. 👍